salt/modules/redismod.py

```diff
@@ -395,7 +395,14 @@ def hmset(key, **fieldsvals):
     database = fieldsvals.pop('db', None)
     password = fieldsvals.pop('password', None)
     server = _connect(host, port, database, password)
-    return server.hmset(key, fieldsvals)
+    # orchestration passes several private key/var pairs in that we don't want
+    # to set in redis.
+    redispairs = {}
+    for field, value in fieldsvals.iteritems():
+        if not field.startswith('__'):
+            redispairs[field] = value
```

Comment:
You don't need to do this, use `salt.utils.args.clean_kwargs`

```
return server.hmset(key, __utils__['args.clean_kwargs'](**fieldvals))
```

Reply:
That works in 2018.3, but I'm working from 2017.7 and `salt.utils.args.clean_kwargs` isn't available there...

Reply:
it will be in `salt.utils.clean_kwargs` in 2017.7, so you cannot use the __utils__ syntax, but you should still use this builtin.

Reply:
Excellent, updating now.